app/screens/edit_profile/edit_profile.js

```diff
@@ -87,16 +88,29 @@ export default class EditProfile extends PureComponent {
         props.navigator.setOnNavigatorEvent(this.onNavigatorEvent);
         props.navigator.setButtons(buttons);
 
+        const profileImageUri = '';
+
         this.state = {
             email,
             firstName,
             lastName,
             nickname,
             position,
             username,
+            profileImageUri,
         };
     }
 
+    getProfileImageUri = async (id, lastPictureUpdate) => {
+        let uri;
+        try {
+            uri = await Client4.getProfilePictureUrl(id, lastPictureUpdate);
```

Comment:
Why are we using the client here directly? In any case this should be an action

Reply:
that's what i wondered. i can refactor this. the main reason is to get the profile image URL because I don't have it in the currentUser object from what i can see. as i mentioned on the other comment, there would be no need to show the "Remove Image" option if the user profile has the default image set.

Reply:
shouldn't 'uploadProfileImage' be an action as well?

Reply:
actually seems that we do not have a selector for this, so I guess this is fine, in any case we could improve this later on, not needed for this PR and perhaps take a look at https://github.com/mattermost/mattermost-mobile/blob/master/app/components/profile_picture/profile_picture.js#L60 as you may use a local cache for the image instead of performing another request

Reply:
This isn't actually making a client call. It's just a helper method to construct the URL that happens to be on the Client4 instance. It could be moved into the utils or something, but we've kept all the URL construction logic in the client for a long time, so I wouldn't worry about changing it now